packages/smooth_app/lib/pages/user_management/forgot_password_page.dart

```diff
@@ -109,7 +109,7 @@ class _ForgotPasswordPageState extends State<ForgotPasswordPage> {
                       style: theme.textTheme.bodyText2,
                     ),
                   const Spacer(flex: 2),
-                  if (_message != 'Error') ...<Widget>[
+                  if (_message != '') ...<Widget>[
```

Comment:
It checks if the error message is not equal to the start value to determine if a error box should be shown, just saves us an extra bool for that. 
Forgot about the direct access to the variable during a code review of the PR